games/hidden_maze.js

```diff
@@ -0,0 +1,235 @@
+/*
```

Comment:
this version is already in the gallery, right? Then delete it so the file doesn't get duplicated

Reply:
Should be deleted, this is only my second time making a pull request, and only my second time making a 2nd pull request from the same fork.

Reply:
no problem!:)